aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+const Error = ({ textError, ...rest }) => {
+  return (
+    <Box {...rest} align="center" gap="xsmall" direction="row">
+      <CircleAlert size="small" />
+      <Text size="xsmall">{textError}</Text>
+    </Box>
+  );
+};
+
+Error.propTypes = {
+  textError: PropTypes.string,
+};
+
+export const RequiredFieldsExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    name: 'Enduro',
+    superPower: 'Sky Runner',
+    nemesis: true,
+    email: 'enduro@skyrunner.io.com',
+    weakness: 'PB & J',
+  });
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Text size="xxlarge" weight="bold">
+            Form Header
+          </Text>
+        </Header>
+        <Box
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            messages={{
+              required: (
+                <Error
+                  background="background-front"
+                  textError="This is a required field."
+                />
+              ),
+            }}
+            value={formValues}
+            onChange={onFormChange}
+          >
+            <RequiredFormField
+              required
+              error={
+                <Error
+                  background="background-front"
+                  textError="Invalid credentials."
+                />
```

Comment:
This might seem nit picky, but since these are being used as code samples, from a style and simplicity standpoint, I'd think we are trying to avoid unnecessarily creating props. We can make this more generic and less opinionated by passing through children.
```suggestion
                <Error>This is a required field.</Error>
```